Gopkg.toml

```diff
@@ -0,0 +1,63 @@
+# Gopkg.toml example
+#
+# Refer to https://golang.github.io/dep/docs/Gopkg.toml.html
```

Comment:
Should we preserve this example code? Is that helpful?

Reply:
Not particularly helpful. I'll remove it.